src/pages/ReactionPage/ReactionPage.tsx

```diff
@@ -21,15 +21,15 @@ import "./ReactionPage.scss";
 const ReactionPage = () => {
   const venueId = useVenueId();
   const venue = useSelector(currentVenueSelectorData);
-  const partygoers = usePartygoers();
+  const { recentVenueUsers } = useRecentVenueUsers();
   const reactions = useSelector((state) => state.firestore.ordered.reactions);
   const chats = useVenueChats(venueId);
   const filteredChats = useMemo(
     () => chats?.filter((chat) => chat.deleted !== true) ?? [],
     [chats]
   );
 
-  const hasPartygoers = useMemo(() => partygoers.length > 0, [partygoers]);
+  const hasPartygoers = recentVenueUsers.length > 0;
```

Comment:
This can be also used like this `!!recentVenueUsers.length`, not a change request just mentioning it.

Reply:
My preference in things like that has been to leaning towards explicitness (where it makes sense to) over coercing things with `!!`